### DISCUSSION TITLE
Fleet如何配置？插件添加完不生效报错

### DISCUSSION THREAD

**Discussion by pengwangpengwang** [izhangzhihao/intellij-rainbow-brackets#2780] [Q&A] [answered]
Rainbow Brackets: You may need to re-create a editable copy of your current theme by action: Edit Color Theme...(And delete current theme)

**Comment by izhangzhihao** [comment, marked as answer]
https://github.com/izhangzhihao/intellij-rainbow-brackets/blob/2022.3/Fleet.md#install-rainbow-brackets-into-your-fleet-ide-and-config-it
